jobs/job_utils.py

```diff
@@ -64,9 +64,36 @@ def clone_model(model, **new_values):
     return cls(id=model_id, **props)
 
 
+def get_model_class(kind):
+    """Returns the model class corresponding to the given kind.
+
+    NOTE: A model's kind is usually, but not always, the same as a model's class
+    name. Specifically, the kind is different when a model overwrites the
+    _get_kind() class method. Although Oppia never does this, the Apache Beam
+    framework uses "kind" to refer to models extensively, so we follow the same
+    convention and take special care to always return the correct value.
+
+    Args:
+        kind: str. The model's kind.
+
+    Returns:
+        type(datastore_services.Model). The corresponding class.
+
+    Raises:
+        KindError. Internally raised by _lookup_model when the kind is invalid.
+    """
+    return datastore_services.Model._lookup_model(kind) # pylint: disable=protected-access
+
+
 def get_model_key(model):
     """Returns the given model's key.
 
+    TODO(#11475): Delete this function after we can use the real datastoreio
+    module. Until then, we need to maintain this code so we can test queries.
+    We need this because NDB queries that target every model can only be
+    performed when we sort models by key, and we use this function to acquire
+    the key for an NDB model.
+
```

Comment:
Addresses https://github.com/oppia/oppia/pull/12880#discussion_r638268287:
> Should we explain here why we need these methods? I imagine getting to this code year from now without any context it might be confusing why exactly is this code needed.